code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -52,14 +52,25 @@
 	required_catalysts = list(/datum/reagent/stabilizing_agent = 1)
 	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_UNIQUE
 
+#define MAX_EXO_TEMP 1064
+
 /datum/chemical_reaction/plasma_solidification
 	required_reagents = list(/datum/reagent/iron = 5, /datum/reagent/consumable/frostoil = 5, /datum/reagent/toxin/plasma = 20)
 	mob_react = FALSE
 	reaction_flags = REACTION_INSTANT
 	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_UNIQUE | REACTION_TAG_OTHER
 
 /datum/chemical_reaction/plasma_solidification/on_reaction(datum/reagents/holder, datum/equilibrium/reaction, created_volume)
-	new /obj/item/stack/sheet/mineral/plasma(get_turf(holder.my_atom), round(created_volume))
+	var/turf/reaction_location = get_turf(holder.my_atom)
+	var/obj/nugget = new /obj/item/stack/nugget/plasma(reaction_location, round(created_volume))
+
+	for(var/creation_loop in 1 to round(created_volume))
+		var/datum/gas_mixture/enviro = reaction_location.return_air()
+		enviro.temperature = max(T20C, enviro.temperature * 1.1)
```

Comment:
This is weird and doesn't make much sense, the heating should never be exponential, otherwise you won't like what players will make with this. Make this a static value